spec/frameSpec.js

```diff
@@ -0,0 +1,47 @@
+describe('Frame', function() {
+
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('when playing', function() {
+
+    it('can receive a bowl', function() {
```

Comment:
In general, make sure the wording of your tests is consistent with the expectation.  Here, you are testing 'can receive a bowl' when in actual fact, the expectation is that following the very first bowl, `frame.firstBowl` is equal to the bowled value.